contracts/HolographFactory.sol

```diff
@@ -327,6 +327,15 @@ contract HolographFactory is Admin, Initializable, Holographable, HolographFacto
     if (v < 27) {
       v += 27;
     }
+    if (v != 27 && v != 28) {
+      return false;
+    }
```

Comment:
Could you add a note about this logic and hashes. 

I found some documentation that looks related in the OZ repo, but this is a bit different